.github/PULL_REQUEST_TEMPLATE/new_entropy_template.md

```diff
@@ -0,0 +1,45 @@
+## Contributor guidelines
+
+- [ ] I verify that I have read the
+    [devdocs](https://juliadynamics.github.io/Entropies.jl/dev/devdocs/).
```

Comment:
I would go for something less formal, and perhaps even less intimidating. E.g., without a checkmark, and saying, something "have a look at the  [devdocs](https://juliadynamics.github.io/Entropies.jl/dev/devdocs/) before finishing your PR."